binding/python3/__init__.py

```diff
@@ -33,9 +38,6 @@ def __setitem__(self, key, value):
     def __delitem__(self, key):
         del self._storage[key]
 
-    def __iter__(self):
```

Comment:
Is there any reason for removing this?

Reply:
I believe it was exactly the same in the parent class.